server/utils/nunjucksSetup.ts

```diff
@@ -340,6 +345,44 @@ export function registerNunjucks(app?: express.Express): Environment {
     return word[0].toUpperCase() + word.substr(1).toLowerCase()
   })
 
+  njkEnv.addFilter('createOffenderLink', (licence: FoundProbationRecord): string => {
+    const isTimedOutLicence = licence?.licenceStatus === 'TIMED_OUT'
+    const isHardStopLicence = licence?.kind === 'HARD_STOP'
+
+    if (isTimedOutLicence && licence.versionOf) {
+      return `/licence/create/id/${licence.licenceId}/licence-changes-not-approved-in-time`
+    }
+
+    if (
+      (isTimedOutLicence && !isHardStopLicence) ||
```

Comment:
you're never going to get a timed out licence that is a hardstop licence so can remove the `!isHardStopLicence`